### PR TITLE
Update new member resources to help with onboarding

### LINKED ISSUE #97: For new members, update info on tools CfD uses
As a new user, I would like to be able to sign in to this app and see an overview of the tools that Code for Denver uses, so I can get up to speed on using those tools quickly.

### PR DESCRIPTION
**Pull request by wdoug**
**NOTE: This pull request is based on top of #123** We'll want to merge that one first

This PR closes #97

#### What does this PR do?
- Adds an initial markdown renderer component
- Updates the new member resources document and converts it to markdown for easier updating
- Adds separation of logged-in routes and logged-out routes
- Makes new landing page for when the user hasn't logged in

Hopefully this will help in our quest to automate onboarding

### CODE REVIEW

**Comment by wdoug** [comment]
This can be seen [here](https://deploy-preview-124--cfd-members.netlify.com/). Make sure you copy and past in the url `https://deploy-preview-124--cfd-members.netlify.com/` before the `#` right after you log in to be able to log in to the actual review app.

**Comment by eemshi** [comment]
hey @wdoug, just fyi, pasting `https://deploy-preview-124--cfd-members.netlify.com/` before the `#` in the url right after logging in didn't do anything for me (still not showing me as logged in)

edit: it worked after a couple of tries and logging in several times. i think the same thing happened with the login the last time i looked at this (last month?) so maybe it's already logged as a separate bug?

ok, just tried again a couple times in new incognito browsers and it seems to be working. must've been a fluke the first time.

File: src/markdown/member_resources.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- [Google Drive](https://drive.google.com/drive/folders/0B15HLk4_JV3nWjkyOGtFUmhKZDQ?usp=sharing)
+  - Collaborative document sharing
+- [GitHub](https://github.com/codefordenver)
+  - Collaborative coding
+- [Waffle](https://waffle.io/codefordenver)
+  - Task management built on top of github issues
+- [Flowdock](https://www.flowdock.com/app/cfa-brigades/code-for-denver)
+  - Chat tool
+  - Requires an invite from and admin to our organization
```
Comment (eemshi) [inline src/markdown/member_resources.md]: tiniest typo: "an admin of"

**Comment by wdoug** [comment]
@eemshi, in regards to the log in issue, there is a known funky situation when a user logs in or signs up. I was thinking it was documented in #91, but I think that might be something else so I went ahead and created a new issue: #127

File: src/utils/MarkdownRenderer.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import marked from 'marked';
+import LoadingIndicator from '../presentational/LoadingIndicator';
+import './MarkdownRenderer.css';
+
+marked.setOptions({
+  gfm: true,
+  sanitize: true // DO NOT CHANGE - helps avoid XSS attack possibility
+});
+
+async function getMarkdownHTML(fileName) {
```
Comment (dviramontes) [inline src/utils/MarkdownRenderer.js:12]: Nice async await function!

**Review by dviramontes — APPROVED** [review]
lgtm!

**Pull request merged**